adafruit_requests.py

```diff
@@ -762,17 +789,33 @@ def wrap_socket(
         return _FakeSSLSocket(socket, self._iface.TLS_MODE)
 
 
+def create_fake_ssl_context(
```

Comment:
needed to break out `create_fake_ssl_context` to beable to pre-create a `SocketHandler` when using and `AirLift` or such via `adafruit_esp32spi`